src/lib/modules/Document/DocumentStats.js

```diff
@@ -27,7 +27,7 @@ export class DocumentStats extends Component {
     const pidType = recordToPidType(document);
     try {
       this.cancellableFetchStats = withCancel(
-        statsApi.recordStats(pidType, document.pid)
+        statsApi.recordStats(pidType, document.metadata.pid)
```

Comment:
how was it working before? 😱

Reply:
because of this: https://github.com/inveniosoftware/react-invenio-app-ils/pull/208/files#diff-34d9d992c176e0090a1cd696e93660a8R181
I prefer to keep the structure of the original object though, it is more readable IMO